src/test/groovy/com/cinnober/gradle/semver_git/SemverGitPluginTest.groovy

```diff
@@ -43,7 +43,22 @@ class SemverGitPluginTest extends GroovyTestCase {
         testParseVersion("1.2.3-SNAPSHOT", [1,2,3,"SNAPSHOT"]);
     }
     void testParseVersion12_34_56_rc78() {
-        testParseVersion("12.34.56-rc78", [12,34,56,"rc78"]);
```

Comment:
The test name doesn't match the other tests, please restore a copy of the old one or rename the test